sesman/env.c

```diff
@@ -200,3 +201,45 @@ env_set_user(const char *username, char **passwd_file, int display,
 
     return error;
 }
+
+
+/******************************************************************************/
+int DEFAULT_CC
+env_add_xauth_user(int display, char *cookie, char *file)
+{
+    FILE *dp;
+    char xauth_str[256];
+    int fd;
+
+    if ( file == NULL )
+    {
+        if (!g_file_exist(".Xauthority")) 
+        {
+            fd = g_file_open(".Xauthority");
+            g_file_close(fd);
```

Comment:
I didn't notice it does create the file, the message on stdout was misunderstood. I'll remove all this part